openpmd_viewer/openpmd_timeseries/data_reader/data_reader.py

```diff
@@ -0,0 +1,242 @@
+"""
+This file is part of the openPMD-viewer
+
+It routes the calls to the data reader to either
+the h5py data reader, or to openpmd-api.
+
+Copyright 2020, openPMD-viewer contributors
+Authors: Remi Lehe
+License: 3-Clause-BSD-LBNL
+"""
+from . import h5py_reader
+backend = 'h5py'
```

Comment:
In the future, there will be some code here that checks whether the `openpmd-api` is available, and will set `backend` to `openpmd-api` in this case.